erizo/src/erizo/rtp/QualityManager.cpp

```diff
@@ -259,9 +262,10 @@ void QualityManager::forceLayers(int spatial_layer, int temporal_layer) {
   temporal_layer_ = temporal_layer;
 }
 
-void QualityManager::setMinDesiredSpatialLayer(int spatial_layer) {
-  ELOG_DEBUG("message: setting min desired spatial layer, spatial_layer: %d", spatial_layer);
-  min_desired_spatial_layer_ = spatial_layer;
+void QualityManager::enableSlideShowBelowSpatialLayer(int spatial_layer) {
+  ELOG_DEBUG("message: Will activate slideshow when below spatial layer, spatial_layer: %d", spatial_layer);
+  slideshow_below_spatial_layer_ = spatial_layer;
+  freeze_fallback_active_ = false;
```

Comment:
is there a way to disable this feature afterwards?

Reply:
It is possible but it's a little bit awkward, you would have to do `enableSlideShowBelowSpatialLayer(-1)` a possible solution would be to have a `disableSlideShowBelowSpatialLayer` either in the client side and have it be a call to this function and add -1 as a parameter or create the whole call stack from the client to a new `QualityManage::disableSlideShowBelowSpatialLayer()`.
Option 1 adds less noise but Option 2 is clearer. What do you think?
Besides, there's a bug currently that may prevent this from working but I'm on it and should be fixed in whatever option we choose.

Reply:
Addressed this in the new commit 👍